sources/packages/web/src/types/AppRoutes.ts

```diff
@@ -47,6 +47,7 @@ export enum AppRoutes {
   SearchInstitutions = "search-institutions",
   InstitutionDetail = "institution-detail/:institutionId",
   AESTStudentProfile = "student-profile",
+  AESTEditInstitutionLocation = "edit-institution-location/institution/:institutionId/location/:locationId",
```

Comment:
Do we have other similar routes?
I would say that the route would be more like the below:
```typescript
institution-location/institution/:institutionId/location/:locationId/edit
```